File: Modul-4/Soal-5/PRAK405-2410817210008-AhmadUlyani.py
n, k = map(int, input().split())

total = 0

for i in range(1, n + 1):
    line_sum = sum(j * k for j in range(i, 0, -1))
    total += line_sum
    terms = ' + '.join(f'({j} * {k})' for j in range(i, 0, -1))
    print(f"{terms} = {line_sum}")

print(total)

File: Modul-4/Soal-5/PRAK405-2410817210008-AhmadUlyani.c
#include <stdio.h>

int main() {
    int n, k;
    scanf("%d %d", &n, &k);
    
    int total = 0;
    for (int i = 1; i <= n; i++) {
        int line_sum = 0;
        for (int j = i; j >= 1; j--) {
            line_sum += j * k;
            printf("(%d * %d)", j, k);
            if (j != 1) printf(" + ");
        }
        printf(" = %d\n", line_sum);
        total += line_sum;
    }
    printf("%d\n", total);
    return 0;
}